src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -207,6 +207,15 @@ export class CalcitePickListItem {
     );
   }
 
+  renderSecondaryAction() {
+    const hasSecondaryAction = this.el.querySelector(`[slot=${SLOTS.secondaryAction}]`);
```

Comment:
`this.el.querySelector(`[slot=${SLOTS.secondaryAction}]`)` is very common now it seems. Maybe we should have a utility for this @jcfranco? `hasSlot(el: HTMLElement, slotName:string): boolean;`?

Reply:
That can be arranged! 😄

Reply:
https://github.com/Esri/calcite-app-components/issues/784

Reply:
Noice!